src/script/ceph-release-notes

```diff
@@ -105,9 +105,9 @@ def split_component(title, gh, number):
         issue_labels = {it['name'] for it in issue['labels']}
         if 'documentation' in issue_labels:
             return 'doc: ' + title
-        item = labels.intersection(issue_labels)
+        item = (labels.union(set(prefixes))).intersection(issue_labels)
```

Comment:
i am not sure why we want to `union(labels, prefixes)` here? if `issue_labels` are all labels attached to the PR in question, `issue_labels` should be a subset of `labels` as long as the latter is updated. so the return value of `intersection()` should always be `issue_labels`.

Reply:
@theanalyst ping?

Reply:
updated, sorry for letting this rot